content/docs/extensions/pg_prewarm.md

```diff
@@ -0,0 +1,241 @@
+The output of `SELECT pg_prewarm()` is the number of blocks from the specified table that was loaded into the Postgres buffer cache. The default block size in Postgres is 8192 bytes (8KB).
+
+The `pg_prewarm` function does not support specifying multiple table names in a single command. It's designed to work with a single table at a time. If you want to prewarm multiple tables, you would need to call `pg_prewarm` separately for each.
+
+## Running pg_prewarm on indexes
+
+Running `pg_prewarm` on frequently-used indexes can help improve query performance after a Postgres restart. You might also run `pg_prewarm` on indexes that are not frequently used but will be involved in upcoming heavy read operations.
+
+Running `pg_prewarm` on an index is similar to running it on a table, but you specify the index's OID (Object Identifier) or its fully qualified name (schema name plus index name) instead. 
+
+Here's an example that demonstrates how to use `pg_prewarm` to preload an index into memory:
+
+```sql
+SELECT pg_prewarm('schema_name.index_name');
+```
+
+Replace `schema_name.index_name` with the actual schema and index name you want to prewarm. If you're not sure about the index name or want to list all indexes for a specific table, you can use the `pg_indexes` view to find out. Here's how you might query for index names:
+
+```sql
+SELECT indexname FROM pg_indexes WHERE tablename = 'your_table_name';
+```
+Replace `your_table_name` with the name of the table whose indexes you're interested in. Once you have the index name, you can then use `pg_prewarm` as shown above.
+
+Additionally, if you prefer to use the index's OID, you can find it using the `pg_class` system catalog. Here's how to find an index's OID:
+
+```sql
+SELECT oid FROM pg_class WHERE relname = 'index_name';
+```
+
+Then, you can use the OID with `pg_prewarm` like so:
+
+```sql
+SELECT pg_prewarm(your_index_oid);
+```
+
+## Check the proportion of a table loaded into cache
+
+In this example, you create a table, check its data size, run `pg_prewarm`, and then check to see how much of the table's data was loaded into memory.
+
+1. First, create a table and populate it with some data:
+
+    ```sql
+    CREATE TABLE t_test AS
+    SELECT * FROM generate_series(1, 1000000) AS id;
+    ```
+
+2. Check the size of the table:
+
+    ```sql
+    SELECT pg_size_pretty(pg_relation_size('t_test')) AS table_size_pretty,
+        pg_relation_size('t_test') AS table_size_bytes;
+    ```
+
+    This command returns the size of the table in both MB and bytes.
+
+    ```sql
+     table_size_pretty | table_size_bytes 
+    -------------------+------------------
+    35 MB              |         36700160
+    ```
+
+3. Load the table data into the Postgres buffer cache using `pg_prewarm`:
+
+    ```sql
+    SELECT pg_prewarm('public.t_test') AS blocks_loaded;
+    ```
+
+    This will output the number of blocks that were loaded:
+
+    ```sql
+    blocks_loaded 
+    ---------------
+            4480
```

Comment:
I got 4425 for some reason

Reply:
Ah, you explain some possible reasons below. :)

Reply:
Checked again to make sure.